src/config.rs

```diff
@@ -2,44 +2,49 @@ use crate::net::constants::{DEFAULT_MTU, FRAGMENT_SIZE_DEFAULT, MAX_FRAGMENTS_DE
     pub max_packet_size: usize,
-    /// These are the maximal fragments a packet could be divided into.
+    /// Value which can specify the maximal allowed fragments.
     ///
     /// Why can't I have more than 255 (u8)?
     /// This is because you don't want to send more then 256 fragments over UDP, with high amounts of fragments the chance for an invalid packet is very high.
     /// Use TCP instead (later we will probably support larger ranges but every fragment packet then needs to be resent if it doesn't get an acknowledgement).
     ///
-    /// Recommended value: 16 but keep in mind that lower is better.
+    /// default: 16 but keep in mind that lower is better.
     pub max_fragments: u8,
-    /// This is the size of a fragment.
-    /// If a packet is too large it needs to be split in fragments.
+    /// Value which can specify the size of a fragment.
     ///
-    /// Recommended value: +- 1450 (1500 is the default MTU)
+    /// use-case: If a packet is too large it needs to be split into fragments.
+    /// default: +- 1450 (1500 is the default MTU)
     pub fragment_size: u16,
-    /// This is the size of the buffer that queues up fragments ready to be reassembled once the whole packet arrives.
+    /// Value which can specify size of the buffer that queues up fragments ready to be reassembled once the whole packet arrives.
     pub fragment_reassembly_buffer_size: usize,
-    /// This is the size of the buffer the UDP socket reads it data into.
+    /// Value which can specify size of the buffer the UDP-bytes will be read into.
+    ///
+    /// default: +- 1450 (1500 is the default MTU)
     pub receive_buffer_max_size: usize,
-    /// This is the factor which will smooth out network jitter. So that if one packet is not arrived fast we don't wan't to directly transform to an bad network.
+    /// Value which can specify the factor which will smooth out network jitter.
     ///
-    /// Recommended value: 10% of the rtt time.
-    /// Value is a ratio (0 = 0% and 1 = 100%)
+    /// use-case: If one packet hast not arrived we don't directly want to transform to a bad network state.
+    /// default: 10% of the rtt time (ratio (0 = 0% and 1 = 100%)
     pub rtt_smoothing_factor: f32,
-    /// This is the maximal round trip time (rtt) for packet.
+    /// Value which can specify the maximal round trip time (rtt) for packet.
     ///
-    /// Recommend value: 250 ms
-    /// Value is represented in milliseconds.
+    /// default: 250 ms
     pub rtt_max_value: u16,
-    /// This is the size of the event buffer we read socket events from `mio::Poll` into.
+    /// Value which can specify the event buffer we read socket events into.
+    ///
+    /// default: 1024
     pub socket_event_buffer_size: usize,
-    /// Optional duration specifying how long we should block polling for socket events.
+    /// Value which can specify how long we should block polling for socket events.
+    ///
+    /// default: 1 ms
```

Comment:
```suggestion
    /// Value that specifies how long we should block polling for socket events, in milliseconds. Defaults to `1ms`.
```
Should probably add in what is being polled.